scalafmt-tests/src/test/resources/scala3/OptionalBraces_keep.stat

```diff
@@ -3626,8 +3626,8 @@ object a {
   implicit def generate[T](
       value: T
   ): Clue[T] = macro MacroCompatScala2.clueImpl
-  implicit def generate[T](value: T): Clue[T] = macro
```

Comment:
I would expect keep to not change the newlines here,

Reply:
and now, after the fix, it **does** preserve the newline :)

Reply:
ach, right my bad. My brain is not working 100% today :sweat_smile: